cmd/crowdsec-cli/decisions.go

```diff
@@ -504,46 +576,50 @@ func NewDecisionsImportCmd() *cobra.Command {
 duration,scope,value
 24h,ip,1.2.3.4
 
-cscsli decisions import -i decisions.csv
+cscli decisions import -i decisions.csv
```

Comment:
we should update the help to:
 - mention it supports stdin with `-i i`
 - mention is support the "plain" format that is simply the ip, one per line